lib/helper/WebDriver.js

```diff
@@ -1007,11 +1004,11 @@ class WebDriver extends Helper {
    * * *Appium*: supported
    */
   async dontSeeElement(locator) {
-    const res = await this._locate(locator, false);
+    const res = await this._locate(withStrictLocator(locator), true);
```

Comment:
I enabled smartWait. In `SmartWait` tests it equals 3 sec

On test page it appears in 1 or 2 sec.
So, method finds element and fails.